learning_assistant/platform_imports.py

```diff
@@ -57,3 +57,39 @@ def get_cache_course_run_data(course_run_id, fields):
     # pylint: disable=import-error, import-outside-toplevel
     from openedx.core.djangoapps.catalog.utils import get_course_run_data
     return get_course_run_data(course_run_id, fields)
+
+
+def learning_assistant_available(course_key):
+    """
+    Return whether the Learning Assistant is available in the course represented by the course_key.
+
+    Note that this may be different than whether the Learning Assistant is enabled in the course. The value returned by
+    this fuction represents whether the Learning Assistant is available in the course and, perhaps, whether it is
+    enabled. Course teams can disable the Learning Assistant via the LearningAssistantCourseEnabled model, so, in those
+    cases, the Learning Assistant may be available and disabled.
+
+    Arguments:
+        * course_key (CourseKey): the course's key
+
+    Returns:
+        * bool: whether the Learning Assistant feature is available
+    """
+    # pylint: disable=import-error, import-outside-toplevel
+    from lms.djangoapps.courseware.toggles import learning_assistant_is_active
```

Comment:
Thinking out loud. Does this create a cms -> learning -> lms import loop and is that a problem in any way?

Reply:
That's a good question. I don't think so. Where do you think the loop would be? I've tried tracing it out, but I can't think of where there might be a loop currently.

I haven't seen any issues when testing the `CourseApps` API in the CMS or when testing the Pages & Resources page.

Some other plugins are defined in the `lms` directory (e.g. [ProctoringCourseApp](https://github.com/openedx/edx-platform/blob/1f3d23250926751acb80aaba6a6bc3b4d7ac3f42/lms/djangoapps/courseware/plugins.py#L167)), so I'd imagine they would have a similar problem, if I'm understanding you correctly.

Reply:
ah I think a 'loop' might be the wrong term here. It's definitely weird to me we're importing from the LMS into the CMS tho. I'd expect shared code to exist in /openedx or /common. I guess if we're already following an established pattern for CourseApps best not to break it however.